js/opentok-hardware-setup.js

```diff
@@ -295,6 +295,11 @@ var authenticateForDeviceLabels = function(callback) {
 
 function createOpentokHardwareSetupComponent(targetElement, options, callback) {
 
+  if (typeof targetElement === 'string') {
+    var targetElementStr = targetElement;
```

Comment:
Or you could just remove this line and make the next line this:

targetElement = document.getElementById(targetElement);